### PR TITLE
[feat] Add read to events

### PR DESCRIPTION
**Pull request by MikeVedsted**
|Read events | route |  controller | service |
|-------|-------|-------|-------|
|all| X | X | X |
|byId| X | X | X |
|byCategory| X | X | X |

### CODE REVIEW

File: src/controllers/event.ts [hunk 1]
```diff
@@ -21,10 +21,25 @@ export const findEventById = async (
   res: Response,
   next: NextFunction
 ) => {
+  const { eventId } = req.params
+  console.log(eventId)
   try {
```
Comment (MarinaFroes) [inline src/controllers/event.ts:26]: Please, remove `console.log()`.

**Review by MarinaFroes — APPROVED** [review]
It seems good.

File: src/controllers/event.ts (hunk 1 above)
Comment (MikeVedsted) [reply src/controllers/event.ts]: D**, thought I had them all 😄

**Review by Rostislav013 — APPROVED** [review]

**Review by MarinaFroes — APPROVED** [review]
Great. 🏅

**Pull request merged**